src/opm/output/eclipse/Summary.cpp

```diff
@@ -280,6 +283,30 @@ namespace {
         return ret;
     }
 
+    std::vector<Opm::EclIO::SummaryNode>
+    requiredAquiferVectors(const std::vector<int>& aquiferIDs)
+    {
+        auto entities = std::vector<Opm::EclIO::SummaryNode> {};
+
+        const auto vectors = std::vector<ParamCTorArgs> {
+            { "AAQR" , Opm::EclIO::SummaryNode::Type::Rate      },
+            { "AAQP" , Opm::EclIO::SummaryNode::Type::Pressure  },
+            { "AAQT" , Opm::EclIO::SummaryNode::Type::Total     },
+            { "AAQTD", Opm::EclIO::SummaryNode::Type::Undefined },
+            { "AAQPD", Opm::EclIO::SummaryNode::Type::Undefined },
+        };
+
+        using Cat = Opm::EclIO::SummaryNode::Category;
+
+        for (const auto aquiferID : aquiferIDs) {
+            for (const auto& vector : vectors) {
+                entities.push_back({ vector.kw, Cat::Aquifer,
+                                     vector.type, "", aquiferID, "" });
```

Comment:
The last argument here is a `std::optional<std::string>` - so passing `{ }` is in my opinion better than an empty string.

Reply:
I've pushed an update to this effect.